### DISCUSSION TITLE
Python to PHP

### DISCUSSION THREAD

**Discussion by Nyxvigi** [ShyrenMore/COVID-19-Hospital-Management#2] [Q&A] [answered]
Hi, can i get the PHP version instead of python 😂

**Comment by ShyrenMore** [comment, marked as answer]
I am afraid that won't be possible, we would need to recreate web-app using some PHP framework like Laravel, and that doesn't seem feasible

**Reply by Nyxvigi** [reply]
Alright. Thanks
